app/src/main/res/values/strings.xml

```diff
@@ -108,8 +108,10 @@
     <string name="flightschool">Dart Flight School</string>
     <string name="flightschool_description">Dart hit 1.0 and is now Flight Ready! Developer communities around the world are hosting Dart Flight School events during the month of February 2014. Get your web development wings with Dart and Angular!</string>
     <string name="ioextended_description">Google I/O Extended is about joining the Google conference at your local group.</string>
+    <string name="wtm_description">Women Techmakers is Google\'s global program and brand for women in technology. Kicked off in 2012 by then VP of Google[x] Megan Smith, Women Techmakers is now led by Women in Technology Advocate Natalie Villalobos and a global team of Googlers who are passionate about empowering women in technology through increased visibility, community, and resources.\n\nEach of our summits and meetups will be catered to local communities of women in technology. Summits and meetups around the world may feature hands-on coding workshops, inspiring speakers, and networking opportunities.\n\nAll events will occur within the month of March.</string>

```

Comment:
I was thinking to have scrollable description for landscape.

On Tue, Feb 24, 2015, 6:39 PM Friedger Müffke <notifications@github.com>
wrote:

> In app/src/main/res/values/strings.xml
> <https://github.com/gdg-x/frisbee/pull/215#discussion_r25267371>:
>
> > @@ -108,8 +108,10 @@
> >      <string name="flightschool">Dart Flight School</string>
> >      <string name="flightschool_description">Dart hit 1.0 and is now Flight Ready! Developer communities around the world are hosting Dart Flight School events during the month of February 2014. Get your web development wings with Dart and Angular!</string>
> >      <string name="ioextended_description">Google I/O Extended is about joining the Google conference at your local group.</string>
> > +    <string name="wtm_description">Women Techmakers is Google\'s global program and brand for women in technology. Kicked off in 2012 by then VP of Google[x] Megan Smith, Women Techmakers is now led by Women in Technology Advocate Natalie Villalobos and a global team of Googlers who are passionate about empowering women in technology through increased visibility, community, and resources.\n\nEach of our summits and meetups will be catered to local communities of women in technology. Summits and meetups around the world may feature hands-on coding workshops, inspiring speakers, and networking opportunities.\n\nAll events will occur within the month of March.</string>
>
> This is a bit long (if the screen is small)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gdg-x/frisbee/pull/215/files#r25267371>.
>